diff --git a/target/linux/ramips/base-files/etc/board.d/01_leds b/target/linux/ramips/base-files/etc/board.d/01_leds
index 1806ff2..01e2363 100755
--- a/target/linux/ramips/base-files/etc/board.d/01_leds
+++ b/target/linux/ramips/base-files/etc/board.d/01_leds
@@ -112,9 +112,6 @@ case $board in
 	hlk-rm04)
 		set_wifi_led "rt2800pci-phy0::radio"
 		;;
-	nexx-wt1520)
-		set_wifi_led "rt2800pci-phy0::radio"
-		;;
 	all0239-3g|\
 	hw550-3g)
 		set_usb_led "hw550-3g:green:usb"
diff --git a/target/linux/ramips/base-files/etc/board.d/02_network b/target/linux/ramips/base-files/etc/board.d/02_network
index 765d398..e027b3b 100755
--- a/target/linux/ramips/base-files/etc/board.d/02_network
+++ b/target/linux/ramips/base-files/etc/board.d/02_network
@@ -103,7 +103,6 @@ ramips_setup_interfaces()
 	dir-320-b1 | \
 	dir-615-h1 | \
 	hlk-rm04 | \
-	nexx-wt1520 | \
 	mzk-w300nh2 | \
 	mzk-750dhp)
 		ucidef_set_interfaces_lan_wan "eth0.1" "eth0.2"
diff --git a/target/linux/ramips/base-files/etc/diag.sh b/target/linux/ramips/base-files/etc/diag.sh
index 472da4b..9ad7ccb 100755
--- a/target/linux/ramips/base-files/etc/diag.sh
+++ b/target/linux/ramips/base-files/etc/diag.sh
@@ -63,9 +63,6 @@ get_status_led() {
 	hlk-rm04)
 		status_led="hlk-rm04:red:power"
 		;;
-	nexx-wt1520)
-		status_led="nexx-wt1520:white:power"
-		;;
 	all0239-3g|\
 	hw550-3g)
 		status_led="hw550-3g:green:status"
diff --git a/target/linux/ramips/base-files/lib/ramips.sh b/target/linux/ramips/base-files/lib/ramips.sh
index 0a09225..bb42ace 100755
--- a/target/linux/ramips/base-files/lib/ramips.sh
+++ b/target/linux/ramips/base-files/lib/ramips.sh
@@ -157,9 +157,6 @@ ramips_board_detect() {
 	*"HILINK HLK-RM04")
 		name="hlk-rm04"
 		;;
-	*"Nexx WT1520")
-		name="nexx-wt1520"
-		;;
 	*"HAME MPR-A1")
  		name="mpr-a1"
  		;;
diff --git a/target/linux/ramips/base-files/lib/upgrade/platform.sh b/target/linux/ramips/base-files/lib/upgrade/platform.sh
index 8ff2da6..407c218 100755
--- a/target/linux/ramips/base-files/lib/upgrade/platform.sh
+++ b/target/linux/ramips/base-files/lib/upgrade/platform.sh
@@ -52,7 +52,6 @@ platform_check_image() {
 	hw550-3g | \
 	hg255d | \
 	hlk-rm04 | \
-	nexx-wt1520 | \
 	ip2202 | \
 	m3 | \
 	m4 | \
diff --git a/target/linux/ramips/dts/NEXXWT1520.dts b/target/linux/ramips/dts/NEXXWT1520.dts
deleted file mode 100644
index ab85486..0000000
--- a/target/linux/ramips/dts/NEXXWT1520.dts
+++ /dev/null
@@ -1,108 +0,0 @@
-/dts-v1/;
-
-/include/ "rt5350.dtsi"
-
-/ {
-	compatible = "NEXXWT1520", "ralink,rt5350-soc";
-	model = "Nexx WT1520";
-
-	memory@0 {
-		device_type = "memory";
-		reg = <0x0 0x2000000>;
-	};
-
-	chosen {
-		bootargs = "console=ttyS1,57600";
-	};
-
-	palmbus@10000000 {
-		uart@500 {
-			status = "okay";
-		};
-
-		spi@b00 {
-			status = "okay";
-			m25p80@0 {
-				#address-cells = <1>;
-				#size-cells = <1>;
-				compatible = "gd25q64";
-				reg = <0 0>;
-				linux,modalias = "m25p80", "s25fl064k";
-				spi-max-frequency = <10000000>;
-
-				partition@0 {
-					label = "u-boot";
-					reg = <0x0 0x30000>;
-					read-only;
-				};
-
-				partition@30000 {
-					label = "u-boot-env";
-					reg = <0x30000 0x10000>;
-					read-only;
-				};
-
-				factory: partition@40000 {
-					label = "factory";
-					reg = <0x40000 0x10000>;
-					read-only;
-				};
-
-				partition@50000 {
-					label = "firmware";
-					reg = <0x50000 0x3b0000>;
-				};
-			};
-		};
-	};
-
-	pinctrl {
-		state_default: pinctrl0 {
-			gpio {
-				ralink,group = "jtag";
-				ralink,function = "gpio";
-			};
-		};
-	};
-
-	ethernet@10100000 {
-		mtd-mac-address = <&factory 0x4>;
-	};
-
-	wmac@10180000 {
-		ralink,mtd-eeprom = <&factory 0>;
-	};
-
-	ehci@101c0000 {
-		status = "okay";
-	};
-
-	ohci@101c1000 {
-		status = "okay";
-	};
-
-	gpio-keys-polled {
-		compatible = "gpio-keys-polled";
-		#address-cells = <1>;
-		#size-cells = <0>;
-		poll-interval = <20>;
-	};
-
-	gpio-leds {
-		compatible = "gpio-leds";
-		power {
-			label = "nexx-wt1520:white:power";
-			gpios = <&gpio0 0 1>;
-		};
-	};
-
-        gpio_export {
-                compatible = "gpio-export";
-                #size-cells = <0>;
-                usb-mode {
-                        gpio-export,name = "usb-mode";
-                        gpio-export,output = <0>;
-                        gpios = <&gpio0 18 0>;
-                };
-        };
-};
diff --git a/target/linux/ramips/image/Makefile b/target/linux/ramips/image/Makefile
index 6f7c51c..35057e4 100644
--- a/target/linux/ramips/image/Makefile
+++ b/target/linux/ramips/image/Makefile
@@ -434,14 +434,6 @@ define BuildFirmware/HLKRM04/initramfs
 endef
 Image/Build/Profile/HLKRM04=$(call BuildFirmware/HLKRM04/$(1),$(1),hlk-rm04,HLKRM04,HLK-RM02)
 
-BuildFirmware/NEXXWT1520/squashfs=$(call BuildFirmware/Default4M/squashfs,$(1),$(2),$(3),$(4))
-define BuildFirmware/NEXXWT1520/initramfs
-	$(call BuildFirmware/OF/initramfs,$(1),$(2),$(3),$(4))
-	#mkhilinkfw -e -i$(call imgname,$(1),$(2))-uImage.bin -o $(call imgname,$(1),$(2))-factory.bin;
-endef
-
-Image/Build/Profile/NEXXWT1520=$(call BuildFirmware/NEXXWT1520/$(1),$(1),nexx-wt1520,NEXXWT1520,NEXXWT1520)
-
 Image/Build/Profile/M3=$(call BuildFirmware/Poray4M/$(1),$(1),m3,M3)
 
 Image/Build/Profile/M4=$(call BuildFirmware/PorayDualSize/$(1),$(1),m4,M4)
@@ -606,7 +598,6 @@ define Image/Build/Profile/Default
 	$(call Image/Build/Profile/FREESTATION5,$(1))
 #	$(call Image/Build/Profile/HG255D,$(1))
 	$(call Image/Build/Profile/HLKRM04,$(1))
-	$(call Image/Build/Profile/NEXXWT1520,$(1))
 	$(call Image/Build/Profile/HW550-3G,$(1))
 	$(call Image/Build/Profile/IP2202,$(1))
 	$(call Image/Build/Profile/M3,$(1))
diff --git a/target/linux/ramips/rt305x/profiles/nexx.mk b/target/linux/ramips/rt305x/profiles/nexx.mk
deleted file mode 100644
index 74edce9..0000000
--- a/target/linux/ramips/rt305x/profiles/nexx.mk
+++ /dev/null
@@ -1,17 +0,0 @@
-#
-# Copyright (C) 2013 OpenWrt.org
-#
-# This is free software, licensed under the GNU General Public License v2.
-# See /LICENSE for more information.
-#
-
-define Profile/NEXXWT1520
-	NAME:=NEXX WT-1520
-endef
-
-define Profile/NEXXWT1520/Description
-	Package set for NEXXWT1520
-endef
-
-$(eval $(call Profile,NEXXWT1520))
-